keras_nlp/models/bert/bert_preprocessor.py

```diff
@@ -32,7 +33,7 @@
 
 
 @keras.utils.register_keras_serializable(package="keras_nlp")
-class BertPreprocessor(keras.layers.Layer):
+class BertPreprocessor(Preprocessor):
     """A BERT preprocessing layer which tokenizes and packs inputs.
```

Comment:
As part of this change, I think we should start showing `from_preset()` usages at the top of the example usages in the class level docstring. We just made a similar change with backbones.

So maybe replace the first time we create a `preprocessor` with `preprocessor = keras_nlp.models.BertPreprocessor.from_preset("bert_base_en_uncased")`.

Then lower down, we could show a shorter "bring your own data" preprocessor example.

```
tokenizer = keras_nlp.models.BertTokenizer(
    vocabulary=["[PAD]", "[UNK]", "[CLS]", "[SEP]", "quick", "brown", "fox"],
)
preprocessor = keras_nlp.models.BertPreprocessor(tokenizer)
```

End goals it to show our most common actually API usage front and center!